modules/external-auth-providers.adoc

```diff
@@ -8,6 +8,7 @@
 
 Direct authentication has been tested with the following OpenID Connect (OIDC) identity providers:
 
+* GitLab
```

Comment:
Should it mention its specific existing bug https://issues.redhat.com/browse/OCPBUGS-61649 ?